gremlin-language/src/test/resources/gremlin-values.txt

```diff
@@ -0,0 +1,419 @@
+# numericLiteral
+0
+1
+123
+9999
++1
+-1
++123
+-123
+1_000
+1_000_000
+123L
+123l
+123B
+123b
+123S
+123s
+123N
+123n
+123I
+123i
+-123L
++456S
+-789b
+0x0
+0x1
+0xA
+0xFF
+0x123ABC
++0x123
+-0x456
+0x1_FF
+0xABCl
+0xDEFi
+0x123
+0X123
+00
+01
+07
+0123
++0123
+-0456
+0123_456
+0123l
+0456s
+0.0
+1.0
+123.456
++1.0
+-1.0
++123.456
+-123.456
+1e2
+1.0e2
+1e+2
+1e-2
+1.23e45
++1.23e45
+-1.23e45
++1.23e+45
+-1.23e-45
+1.0f
+1.0F
+1.0d
+1.0D
+1.0m
+1.0M
+1_000.5
+1.234_567
+1_2.3_4e5_6
+1e2
+1E2
+123f
+123F
+123d
+123D
+123m
+123M
++0
+-0
+0.01
+1.00
+2147483647
+-2147483648
+9999999999
+0.0000000001
+# booleanLiteral
+true
+false
+# stringLiteral
+""
+''
+"hello"
+'hello'
+"hello world"
+'hello world'
+"hello\nworld"
+'hello\nworld'
+"hello\tworld"
+'hello\tworld'
+"hello\\world"
+'hello\\world'
+"hello\"world"
+'hello\'world'
+"hello\rworld"
+'hello\rworld'
+"hello\fworld"
+'hello\fworld'
+"hello\bworld"
+'hello\bworld'
+"hello\123world"
+'hello\123world'
+"hello\0world"
+'hello\0world'
+"hello\07world"
+'hello\07world'
+"hello\377world"
+'hello\377world'
+"hello\u0041world"
+'hello\u0041world'
+"hello\uAAAAworld"
+'hello\uAAAAworld'
+"!@#$%^&*()"
+'!@#$%^&*()'
+"symbols + special chars: <>,.?/"
+'symbols + special chars: <>,.?/'
+"mixed 123 abc !@#"
+'mixed 123 abc !@#'
+"line1\
```

Comment:
I'm a bit surprised that this works at all considering ReferenceGrammarTest only parses a single line at a time. This might not be testing as much as it appears.

Reply:
found a bug. there's some bad data in gremlin-values.txt. cleaning that up.